src/intersection/intersection.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Creates an array of values that are in both the first and the second arrays.
+ *
+ * @param {Array} [array=[]] - The array to inspect.
+ * @param {Array} [values=[]] - The values to exclude.
```

Comment:
'exclude' is causing misunderstandings.  If values of the second array is excluded, how the function is intersection? Can be simple as :
<!--StartFragment-->* @param {Array} [array=[]] - first array to inspect. 
*@param {Array} [values=[]] - second array to inspect..<!--EndFragment-->